app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+
+import { Alert, Toastr } from "neetoui";
+
+const DeleteAlert = ({
+  setContacts,
+  onClose,
+  selectedContactId,
+  setSelectedContactId,
+}) => {
+  const [deleting, setDeleting] = useState(false);
```

Comment:
If it's a boolean, in JS side, it's preferred to prefix with an `is`, `has` etc keyword to better communicate that the value is either true or false.